pages/devs/reputers/reputers.mdx

```diff
@@ -1,150 +1,214 @@
+   - The `Token` variable is specific to the endpoint you expose in your `main.py` file. It is **not** related to any blockchain parameter and is only locally specific.
+4. `minStake`: The minimum stake required to participate as a reputer. This stake will be deducted from the reputer's wallet balance.
+
+<Callout>
+When placing your minimum stake, the system will verify the amount of funds you have already staked in the topic. If your staked amount is insufficient, it will automatically pull the necessary funds from your wallet to meet the required minimum.
+</Callout>
+
+<Callout type="warning">
+The `reputer` config is an array of sub-objects, each representing a different topic ID. This structure allows you to manage multiple topic IDs, each within its own sub-object.
+
+To deploy a reputer that provides inferences for multiple topics, you can duplicate the existing sub-object and add it to the `reputer` array. Update the `topicId`, `SourceOfTruthEndpoint`, `minStake` and `Token` fields with the appropriate values for each new topic:
+```json
+"worker": [
+      {
+          "topicId": 1,
+          "reputerEntrypointName": "api-worker-reputer",
+          "loopSeconds": 30,
+          "minStake": 100000,
+          "parameters": {
+              "SourceOfTruthEndpoint": "http://source:8888/truth/{Token}/{BlockHeight}",
+              "Token": "ethereum"
+          }
+      },
+      // reputer providing ground truth for topic ID 2
+      {
+          "topicId": 2,
+          "reputerEntrypointName": "api-worker-reputer",
+          "loopSeconds": 30,
+          "minStake": 100000,
+          "parameters": {
+              "SourceOfTruthEndpoint": "http://source:8888/truth/{Token}/{BlockHeight}",
+              "Token": "ethereum"
+          }
+      }
+    ],
 ```
-docker compose up
-```
+</Callout>
 
-If you're using Apple Silicon hardware, please make sure to use run docker-compose.arm64.yml instead.
+## Create the Truth Server
 
-### Components of the Example Reputer Node
+### Prepare the API Gateway
 
-This example contains a service for sourcing truth -- called the "`truth` service" -- a Blockless worker node for responding to Blockless requests and calculating topic-specific loss functions, and a Blockless head node for initiating requests.
+Ensure you have an API gateway or server that can accept API requests to call your model.
 
-In practice, you would only run a head node as a side car to a validator node. This way, chain validators can initiate requests to collect inferences and forecasts from workers, and ground-truth-informed loss calculations from reputers.
+### Server Responsibilities
 
-You can read more about the lifecycle of Blockless requests [on the Blockless docs](https://blockless.network/docs/protocol/networking).
+- Accept API requests from `main.py`, your custom Python script executed by the node function.
```

Comment:
> "Accept API requests from main.py, your custom Python script executed by the node function."

This is not correct imo. The responsibility is indeed have to "Accept API requests". but not from main.py, but from the allora-offchain-node process (main.go actually).

Please note that once we use an external model endpoint instead of this toy server, there's absolutely no requirement that you have to use `main.py` or even Python (the text may suggest otherwise, as if you would need a "custom Python script" )

Reply:
Understood for the first point. For this point ("Please note that once we use an external model endpoint instead of this toy server, there's absolutely no requirement that you have to use main.py or even Python (the text may suggest otherwise, as if you would need a "custom Python script" )") specifically I think Nick wanted to make the entire flow python friendly since 98% of model creators opt to use python and may not know any Go. So, I was trying to display where model creators would make adjustments to their models in a clear way using python.

Reply:
I wrote "Accept API requests from `main.go`.